### PR TITLE
Benchmark lite obfuscation

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- applied lite obfuscation (skip PEM keys obfuscation)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] Benchmark
- [x] TODO: rollback custom benchmark after https://github.com/Samsung/CredData/pull/54

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -46,7 +47,7 @@ jobs:
 
     - name: Generate Data Asset
       if: steps.cache-data.outputs.cache-hit != 'true'
-      run: python download_data.py --data_dir data
+      run: python download_data.py --data_dir data --light_obfuscation
```
Comment (babenek) [inline .github/workflows/benchmark.yml:50]:
```suggestion
      run: python download_data.py --data_dir data --lite_obfuscation
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -22,7 +22,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        ref: lightobf
+        repository: babenek/CredData
```
Comment (babenek) [inline .github/workflows/benchmark.yml:26]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -62,7 +63,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:67]:
```suggestion
        repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
todo

**Review by xDizzix — APPROVED** [review]
+

**Review by iuriimet — APPROVED** [review]

**Pull request merged**